src/coap_mbedtls.c

```diff
@@ -1106,9 +1131,15 @@ set_ciphersuites(mbedtls_ssl_config *conf, coap_enc_method_t method) {
         }
 #endif /* MBEDTLS_VERSION_NUMBER >= 0x03020000 */
 #if defined(MBEDTLS_KEY_EXCHANGE__SOME__PSK_ENABLED)
+#if MBEDTLS_VERSION_NUMBER >= 0x03060000
```

Comment:
How about we move this inside the `static` wrapper itself, something like:

```
static int coap_ssl_ciphersuite_uses_psk(..)
{
   #if MBEDTLS_VERSION_NUMBER >= 0x03060000
   ...
   #else
        return mbedtls_ssl_ciphersuite_uses_psk(...);
   #endif
}
```

Reply:
Good suggestion - that works for me as it is cleaner.  Please make that change.